### ISSUE TITLE
Summary of TA's review and peer review

### ISSUE THREAD

**Issue by Anthea98** [PANDASANG1231/522_Ramen#45]
This is the summary of places we need to optimize!
### Proposal 
1. Make the question more clear, whether it is classification or regression, and the reason for using logistic regression, and AUC scores.
2. What about data visualization? What specifically are you going to do?
-------do not quite understand this point-----------
3. packages we use in models.
4. PR: Specify the instructions on how to install and activate the environment needed for doing the whole process.
5. PR: the explicit command lines to be run in the terminal
------we have a workflow chart and a make file, is it necessary?-----
6. PR: link to the data source 

### Report
1. Delete the HTML report/the EDA report and need to create an MD file instead.
2. State the motivation about EDA
------(original suggestions: where is your motivation for the things that you have done? How do you wanna handle the missing values? What did you infer from your analysis? Just plotting the results without any results seems a bit pointless.)--------
------maybe some explanations about what we found in the profiling reports.
3. Same writing questions like Proposal-point1
4. PR: Identified that your dataset has a class imbalance, how are you addressing to handle it?
------we don't conduct a predict right?------
5. PR: Address the size of the test set and your model performance.
6. PR: grammatical and formatting errors (Fig 3. caption alignment)
7. PR: Discuss future direction like how you will address the limitations of your research in the future with better data and methods.
8.  PR: Include authors' names.
9. PR: Elaborate more on the coefficients and state how exactly the important features are impacting the classification.

### Models
1. PR: question about the class imbalance
2. PR: a confusion matrix for how your model performed on the test data. (mentioned many times)
3. PR: 5 fold cross validation to assess the model
4. PR: why drop the top-ten column? (maybe clarify it in EDA)

### Scripts
1. The usage for the EDA script does not match the script itself. Its called generate_EDA_figures.py, but the usage states create_EDA_figures.py

**Comment by datallurgy** [comment]
Report RE: EDA:
We used pandas-profiling to determine that the Top Ten column has too many nulls (98.4% of the training set was missing) and that the Review # column was a unique id and they shouldn't be used for modeling. The remainder of the set had very few missing values (just two missing values in the Type column).

I believe the distribution of star ratings helped @PANDASANG1231 determine an appropriate cut off for the classification.

The word cloud was an attempt to visualise common words that might guide selection of higher ranked noodle choices.

There was a comment from a reviewer to possibly include a plot of ramen style with the ratings. This is a pretty simple plot to make and would be interesting to see. @PANDASANG1231 @Anthea98 @shyan0903  Thoughts?

**Comment by Anthea98** [comment]
In the report method part: two sentences conflict with each other:
The class distribution for the two classes is 0.7 vs. 0.3, which is reasonable and not a concern for class imbalance.
Since we have a class imbalance, we use AUC as the scoring metric. 
Please take care of this when optimizing report~~And also delete the five cv part since it seems we don't do this finally @shyan0903

**Comment by Anthea98** [comment]
I think that is nice for our explanation about EDA @datallurgy
